File: Move_Zeroes.java
class Solution {
     public void moveZeroes(int[] nums) {
        int ZeroSize = 0; 
        for (int i=0;i<nums.length;i++){
	        if (nums[i]==0){
                ZeroSize++; 
            }
            else if (ZeroSize > 0) {
	            int temp = nums[i];
	            nums[i]=0;
	            nums[i-ZeroSize]=temp;
            }
        }
    }
}

File: DSA Day 3/moveZeroes.c
void moveZeroes(int* nums, int numsSize) 
{
 int i, j = 0, temp;
 for(i = 0; i<numsSize; i++)
 {
    if(nums[i]!= 0)
    {
        temp = nums[j];
        nums[j] = nums[i];
        nums[i] = temp;
        j++;
    }
 }   
    
}